u2f-ref-code/java/tests/com/google/u2f/key/messages/TransferAccessMessageTest.java

```diff
@@ -0,0 +1,255 @@
+package com.google.u2f.key.messages;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+import com.google.u2f.TestVectors;
+import com.google.u2f.U2FException;
+import com.google.u2f.server.impl.BouncyCastleCrypto;
+import com.google.u2f.server.Crypto;
+
+
+import java.security.cert.X509Certificate;
+
+public class TransferAccessMessageTest extends TestVectors {
+  private static final int SEQUENCE_NUMBER = 1;
+  private static final int SEQUENCE_NUMBER_OTHER = 0;
+  private static final byte[] NEW_USER_PUBLIC_KEY = TRANSFER_ACCESS_PUBLIC_KEY_B_HEX;
+  private static final byte[] NEW_USER_PUBLIC_KEY_OTHER = TRANSFER_ACCESS_PUBLIC_KEY_C_HEX;
+  private static final byte[] APPLICATION_SHA256 = APP_ID_SIGN_SHA256;
+  private static final byte[] APPLICATION_SHA256_OTHER = APP_ID_ENROLL_SHA256;
+  private static final X509Certificate NEW_ATTESTATION_CERTIFICATE = VENDOR_CERTIFICATE;
+  private static final X509Certificate NEW_ATTESTATION_CERTIFICATE_OTHER = TRUSTED_CERTIFICATE_2;
+  private static final byte[] SIGNATURE_USING_AUTHENTICATION_KEY =
+      TRANSFER_ACCESS_MESSAGE_SIGNATURE_USING_AUTHENTICATION_KEY_A_TO_B;
+  private static final byte[] SIGNATURE_USING_AUTHENTICATION_KEY_OTHER = new byte[1];
+  private static final byte[] SIGNATURE_USING_ATTESTATION_KEY =
+      TRANSFER_ACCESS_MESSAGE_SIGNATURE_USING_ATTESTATION_KEY_A_TO_B;
+  private static final byte[] SIGNATURE_USING_ATTESTATION_KEY_OTHER = new byte[1];
+  private static final TransferAccessMessage TRANSFER_ACCESS_MESSAGE = new TransferAccessMessage(
+      SEQUENCE_NUMBER, NEW_USER_PUBLIC_KEY, APPLICATION_SHA256, NEW_ATTESTATION_CERTIFICATE,
+      SIGNATURE_USING_AUTHENTICATION_KEY, SIGNATURE_USING_ATTESTATION_KEY);
+  
+  private final Crypto crypto = new BouncyCastleCrypto();
+
+  @Test
+  public void testEquals() {
+    TransferAccessMessage transferAccessMessage1 = new TransferAccessMessage(SEQUENCE_NUMBER,
```

Comment:
There are lots of different ways to style this, but I personally prefer this type of style for things with lots of parameters:
```
TransferAccessMessage transferAccessMessage1 =
        new TransferAccessMessage(SEQUENCE_NUMBER,
                                  NEW_USER_PUBLIC_KEY,
                                  APPLICATION_SHA256,
                                  NEW_ATTESTATION_CERTIFICATE,
                                  SIGNATURE_USING_AUTHENTICATION_KEY,
                                  SIGNATURE_USING_ATTESTATION_KEY);
```

That makes it easy to read the parameters.